.github/workflows/build_and_test.yml

```diff
@@ -841,7 +841,8 @@ jobs:
           ./vm-builder \
             -spec=compute/vm-image-spec-${{ matrix.version.debian }}.yaml \
             -src=neondatabase/compute-node-${{ matrix.version.pg }}:${{ needs.tag.outputs.build-tag }} \
-            -dst=neondatabase/vm-compute-node-${{ matrix.version.pg }}:${{ needs.tag.outputs.build-tag }}
+            -dst=neondatabase/vm-compute-node-${{ matrix.version.pg }}:${{ needs.tag.outputs.build-tag }} \
+            -size=1200M
```

Comment:
We do need bigger than 1G for pg16 (and presumably pg17), but it may not need to be as much as 1200M. Guess it's time for a bit of binary search, unless you know a way to easily establish how much space the images are currently actually using?

Reply:
It seems we're going to increase it anyway in [another PR](https://github.com/neondatabase/neon/pull/9498#pullrequestreview-2397261135), so I'm ok here